drivers/options/options.go

```diff
@@ -31,7 +31,7 @@ func GetValueFromDriverOptions(driverOptions *types.DriverOptions, optionType st
 				return &value
 			}
 		}
-		return nil
+		return (*bool)(nil)
```

Comment:
Please don't return a bool point nil. This function will return `interface{}` and if we are using `(*bool)(nil)`, the return value of function `GetValueFromDriverOptions` will be not equal to `nil`

Reply:
For [example](https://play.golang.org/p/ee0IyFu2FA9): 
```go
package main

import (
	"fmt"
)

func main() {
	fmt.Println(testBoolPointer() == nil)
}

func testBoolPointer() interface{} {
	return (*bool)(nil)
}
```
And will print:
```
false
```